libs/sdk-backend-tiger/src/backend/uiSettings.ts

```diff
@@ -82,6 +82,8 @@ export const DefaultUiSettings: ISettings = {
     enableReversedStacking: true,
     enableSeparateTotalLabels: true,
 
+    ["msf.enableTenantCustomModel"]: false,
```

Comment:
IMHO this should be defined in the interface of this object as well.

Reply:
I haved updated ISettings interface. Thanks.